crates/hdi/src/hdi.rs

```diff
@@ -36,6 +36,8 @@ pub trait HdiT: Send + Sync {
         &self,
         must_get_agent_activity_input: MustGetAgentActivityInput,
     ) -> ExternResult<Vec<RegisterAgentActivity>>;
+    // DPKI
+    fn is_same_agent(&self, key_1: AgentPubKey, key_2: AgentPubKey) -> ExternResult<bool>;
```

Comment:
I wonder if we actually want to call it `is_same_agent`. Do we want to use "agent" to talk about the pubkey? What is the name we want for the underlying lineage? "Lineage"? `is_same_lineage`?

Reply:
I like that this implies that an agent now transcends its particular keypair. An "agent" is now something that has exactly one keypair associated with it, but that keypair can change over time. The individual keypairs are not "agents". So I don't suggest to change it.